example/pubspec.lock

```diff
@@ -5,49 +5,49 @@ packages:
     dependency: transitive
     description:
       name: async
-      url: "https://pub.dartlang.org"
+      url: "https://pub.flutter-io.cn"
```

Comment:
Hmmm, strange changes…

Reply:
my bad, I changed the souce of pub, it should not be commited.